src/Vogen/Generators/Conversions/GenerateNewtonsoftJsonConversions.cs

```diff
@@ -21,9 +21,12 @@ public string GenerateAnyBody(TypeDeclarationSyntax tds, VoWorkItem item)
             return string.Empty;
         }
 
-        string code =
-            Templates.TryGetForSpecificType(item.UnderlyingType, "NewtonsoftJsonConverter") ??
-            Templates.GetForAnyType("NewtonsoftJsonConverter");
+        string? code =
```

Comment:
Good choice. So we try for a specific type first, and if not, we try for any old type, but differentiate between value/reference type. That makes perfect sense!